modules/monitoring-silencing-alerts.adoc

```diff
@@ -1,30 +1,54 @@
 // Module included in the following assemblies:
 //
-// * monitoring/cluster_monitoring/managing-cluster-alerts.adoc
+// * monitoring/managing-alerts.adoc
 
-[id="monitoring-silencing-alerts_{context}"]
+[id="silencing-alerts_{context}"]
 = Silencing alerts
 
 You can either silence a specific alert or silence alerts that match a specification that you define.
 
+.Prerequisites
+
+* You have access to the cluster as a developer or as a user with `edit` permissions for the project that you are viewing metrics for.
+
 .Procedure
 
-To silence a set of alerts by creating an alert specification:
+To silence a specific alert:
 
-. Navigate to the *Monitoring* -> *Alerting* -> *Silences* page of the {product-title} web console.
+* In the Administrator perspective:
 
-. Click *Create Silence*.
+. Navigate to the *Monitoring* -> *Alerting* -> *Alerts* page of the {product-title} web console.
 
-. Populate the *Create Silence* form.
+. For the alert that you want to silence, select the {kebab} in the right-hand column and select *Silence Alert*. The *Create Silence* form will appear with a prepopulated specification for the chosen alert.
 
-. To create the silence, click *Silence*.
+. Optional: Modify the silence.
 
-To silence a specific alert:
+. You must add a comment before creating the silence.
 
-. Navigate to the *Monitoring* -> *Alerting* -> *Alerts* page of the {product-title} web console.
+. To create the silence, select *Silence*.
+
+* In the Developer perspective:
+
+. Navigate to the *Monitoring* -> *<project_name>* -> *Alerts* page in the {product-title} web console.
 
-. For the alert that you want to silence, click the button in the last column and click *Silence Alert*. The *Create Silence* form will appear with prepopulated specification of the chosen alert.
+. Expand an alert's details by selecting *>* to the left of the alert's name. Select the name of the alert in the expanded view to open the *Alert Details* page for the alert.
+
+. Select *Silence Alert*. The *Silence Alert* form will appear with a prepopulated specification for the chosen alert.
 
 . Optional: Modify the silence.
 
-. To create the silence, click *Silence*.
+. You must add a comment before creating the silence.
+
+. To create the silence, select *Silence*.
+
+. To create the silence, select *Silence*.
```

Comment:
duplicated with Line 42, should be removed

Reply:
Good catch, thank you! Resolved.